commons/get-help/reference-environment-variables.md

```diff
@@ -284,8 +284,8 @@ So you can alter the build&start process for your application.
     <td></td>
   </tr>
   <tr>
-    <td>CC_JAVA_VERSION</td>
-    <td>Choose the JVM version between 'graalvm-ce' or `8`, `11` or `13` for OpenJDK.</td>
+    <td>[CC_JAVA_VERSION](/doc/java/select-java-version/)</td>
+    <td>Choose the JVM version between `graalvm-ce` for GraalVM 11.0 or `7` to `14` for OpenJDK.</td>
```

Comment:
I think it would make more sens to fist have the Java versions and then the graalvm one. Also, I was confused by the `GraalVM 11.0` thing, maybe we could have `or graalvm-ce for GraalVM 20.1.0 based on OpenJDK 11.0`

Reply:
Very good point, I'll change that.